docs/migration/fastapi.rst

```diff
@@ -260,5 +262,4 @@ Middleware
 Pure ASGI middleware is fully compatible, and can be used with any ASGI framework. Middlewares
 that make use of FastAPI/Starlette specific middleware features such as
 Starlette’s `BaseHTTPMiddleware <https://www.starlette.io/middleware/#basehttpmiddleware>`_ are not compatible,
-but can be easily replaced by making use of `AbstractMiddleware
-<usage/7-middleware/2-creating-middleware/2-using-abstract-middleware/>`_
+but can be easily replaced by :doc:`Creating Middlewares </usage/middleware/creating-middleware>`.
```

Comment:
@provinzkraut the docs preview is not up to date it seems - I don't see this change reflected.. can we make a way to re-trigger it?

Reply:
scratch that, I was looking at a bad section